contracts/GenArt721CoreV3.sol

```diff
@@ -834,6 +892,7 @@ contract GenArt721CoreV3 is
         external
         onlyUnlocked(_projectId)
         onlyArtistOrAdminACL(_projectId, this.updateProjectLicense.selector)
+        onlyNonEmptyString(_projectLicense)
```

Comment:
nit: should we also be using `onlyValidProjectId ` since adminACL can call this?

Reply:
See: https://github.com/ArtBlocks/artblocks-contracts/pull/310#discussion_r981381481